com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/PathTracing/Shaders/PathTracingLight.hlsl

```diff
@@ -262,15 +262,37 @@ float3 GetPunctualEmission(LightData lightData, float3 outgoingDir, float dist)
     return emission;
 }
 
-float3 GetDirectionalEmission(DirectionalLightData lightData, float3 outgoingVec)
+float3 GetDirectionalEmission(DirectionalLightData lightData, float3 position)
 {
     float3 emission = lightData.color;
 
+#if SHADEROPTIONS_PRECOMPUTED_ATMOSPHERIC_ATTENUATION
+    // Nothing to do here
+#else
+    // Physical sky emission color code, adapted from EvaluateLight_Directional()
```

Comment:
Sanity check, is this code call for any kind of sky? what's happen with an HDRI sky?

Reply:
It was the same in the original function, I think it comes from the dynamic if() condition right after this (or at least that's what I saw when checking its effect: if the "affect sky" on the light is off, or if there is no physical sky, then the if condition is false).